README.md

```diff
@@ -50,7 +50,7 @@ To use these settings you can either pass them as keyword arguments to `setup` (
 # Direct
 my_prob = ECOS.setup(n, m, ..., c, h, b; maxit=10, feastol=1e-5)
 # MathProgBase (with JuMP)
-m = Model(solver=ECOS.ECOSSolver(maxit=10, feastol=1e-5))
+m = Model(with_optimizer(ECOS.Optimizer, maxit=10, feastol=1e-5))
```

Comment:
It should be named `model` according to the JuMP style guide

Reply:
It was called `m` previously and I didn't want to break the more localized ECOS convention.

Reply:
Feel free to rename everything so that it conforms to the style guide http://www.juliaopt.org/JuMP.jl/v0.19.0/style/

Reply:
`m` is not related to ECOS, it is how we used to name the model everywhere but it should now be `model` :)

Reply:
Updated to meet stylistic requests.